### DISCUSSION TITLE
Problem about MF6 External Files

### DISCUSSION THREAD

**Discussion by Xiangbo000** [modflowpy/flopy#2278] [Q&A]
When I use set_record to output K, the output file automatically breaks lines every 20 data. This way I can't interact with pyemu. The output looks like the following.
![image](https://github.com/user-attachments/assets/8ad9e3de-2cc3-4936-88fd-eeb98e9ad802)

**Comment by wpbonelli** [comment]
@Xiangbo000 could you provide a minimal script reproducing this? thanks

**Reply by jtwhite79** [reply]
I think this is the wrapped format stuff but I cant remember how to turn it off...maybe if external data files are being request, wrapped format could be turned off by default?

**Reply by wpbonelli** [reply]
I'm ignorant re: wrapped format, maybe someone else can chime in here

**Comment by christianlangevin** [comment]
You may be able to set the maximum number of columns that will be used when writing an array by using this syntax, for example:

```
sim.simulation_data.max_columns_of_data = 100
```
